### ISSUE TITLE
NaN VAE training toss when using convolutional encoder and decoder

### ISSUE THREAD

**Issue by darianSmalley** [pycroscopy/atomai#85]
Training a VAE on grayscale image data when the "conv_encoder" and "conv_decoder" arguments of the VAE constructor method are set to True results in NaN values for the reported training loss. Printing the train loss history also shows an array filled with nan values.

I have attached an image which shows an example of the input data and the report training loss:

![VAE nan training loss](https://github.com/pycroscopy/atomai/assets/6811552/f0977b0b-5765-41fc-ad20-93f5773080f1)

**Comment by ziatdinovmax** [comment]
Sometimes this can be due to data being not normalized. Can you try normalizing it or doing z-standardization to see if that helps? Reducing the learning rate may also help sometimes.

**Comment by darianSmalley** [comment]
Normalizing to [0,1] fixed the nan issue. Thanks! 

Also, what is the intended way to change the learning rate?

**Comment by ziatdinovmax** [comment]
You can can do it by passing a custom optimizer:

```python3
optimizer = lambda p: torch.optim.Adam(p, lr=1e-3)  # specify your learning rate
vae.fit(X_train, optimizer=optimizer, training_cycles=100)
```
